lib/core/info-drawer/info-drawer.component.ts

```diff
@@ -22,7 +22,7 @@ import { MatTabChangeEvent } from '@angular/material';
     template: '<ng-template><ng-content></ng-content></ng-template>'
 })
 export class InfoDrawerTabComponent {
-    /** The title of the tab. */
+    /** The title of the tab (string or translation key). */
```

Comment:
There was another PR recently to enable i18n for the labels, and the comments and docs were updated. We don't need this anymore?